src/oc_bifrost/apps/bifrost/priv/pgsql_statements.config

```diff
@@ -53,6 +53,24 @@
       WHERE authz_id=$1
     ) AS exists">>}.
 
+{get_actor_id,
+<<"SELECT id FROM auth_actor WHERE authz_id=$1">>}.
+
+{get_group_id,
+<<"SELECT id FROM auth_group WHERE authz_id=$1">>}.
+
+{get_container_id,
+<<"SELECT id FROM auth_container WHERE authz_id=$1">>}.
+
+{get_object_id,
+<<"SELECT id FROM auth_object WHERE authz_id=$1">>}.
+
+%% Returns every group an actor is a member of recursively,
+%% including parent groups.
+{get_all_recursive_groups_for_actor,
+<<"SELECT authz_id FROM auth_group 
+   WHERE id IN (SELECT groups_for_actor($1))">>}.
```

Comment:
groups_for_actor has a comment above it saying that it takes internal database ids and is not designed to be called from outside.  But we pass $1 directly to it.  Wouldn't it be better to take an authz_id and then use `actor_id($1)`